presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -46,12 +47,14 @@
     private final int queuedDrivers;
     private final int completedDrivers;
     private final int totalDrivers;
+    private final ErrorType errorType;
```

Comment:
Add `ErrorCode` here and add `ErrorType` to `QueryInfo`. It is weird to have one but not the other, and we should make them consistent (and `QueryInfo` should have everything).